src/components/pages/download_page.py

```
@@ -0,0 +1,91 @@
+import streamlit as st
+
+from src.models import TemplateRepository
+from src.router import Page
+from src.schemas import TemplateFormat
+from src.services.template_converter_service import TemplateConverterService
+
+
+@st.dialog("実行確認", width="small", dismissible=True)
+def confirm_execute_dialog():
+    st.write("こちらの実行します\nよろしいですか？")
```

Ruff (0.12.2)

String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)